### PR TITLE
Add xform metadata for forms published by kpi

### LINKED ISSUE #876: Add info to show form was published by formbuilder
This info should be used by zebra to know if it should add a formbuilder edit link for a form or not. At the moment, it only makes sense to have edit links for forms that were published/deployed by the formbuilder.

### PR DESCRIPTION
**Pull request by ivermac**
closes #876

### CODE REVIEW

File: onadata/apps/api/viewsets/project_viewset.py [hunk 1]
```diff
@@ -91,6 +92,13 @@ def forms(self, request, **kwargs):
 
                 serializer = serializer_cls(survey,
                                             context={'request': request})
+
+                published_by_formbuilder = request.data.get(
+                    'published_by_formbuilder'
+                )
+                if published_by_formbuilder == u'True':
```
Comment (denniswambua) [inline onadata/apps/api/viewsets/project_viewset.py]: There is a utility function for parsing Boolean. You should use it here

Comment (ivermac) [inline onadata/apps/api/viewsets/project_viewset.py]: Where do I find it? Also, `published_by_formbuilder` has string value, how will that utility function be helpful?

File: onadata/libs/serializers/project_serializer.py [hunk 2]
```diff
@@ -168,12 +169,22 @@ class Meta:
...
+            'published_by_formbuilder',
             'last_submission_time',
             'date_created',
             'url',
             'last_updated_at'
         )
 
+    def get_published_by_formbuilder(self, obj):
+        md = obj.metadata_set.filter(
+            data_type='published_by_formbuilder'
```
Comment (denniswambua) [inline onadata/libs/serializers/project_serializer.py]: You can use `unique_type_for_form` function from the metadata to retrieve the value. It has implemented this already

**Comment by skambo** [comment]
@denniswambua pls review

**Comment by ivermac** [comment]
Note: The management command should be tested on both stage and production. Testing involves ensuring that forms that were initially deployed using the formbuilder should have an edit url in zebra.

**Comment by skambo** [comment]
@ivermac The edit button doesn't display for existing forms.  Also, the form builder questions added don't display on charts 

screen shot 2017-01-24 at 10 31 22 am

**Pull request merged**